ksql-execution/src/main/java/io/confluent/ksql/execution/plan/TableGroupBy.java

```diff
@@ -15,22 +15,26 @@
 package io.confluent.ksql.execution.plan;
 
 import com.google.errorprone.annotations.Immutable;
+import io.confluent.ksql.GenericRow;
 import io.confluent.ksql.execution.builder.KsqlQueryBuilder;
 import io.confluent.ksql.execution.expression.tree.Expression;
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.streams.kstream.KGroupedTable;
+import org.apache.kafka.streams.kstream.KTable;
 
 @Immutable
-public class TableGroupBy<T, G> implements ExecutionStep<G> {
+public class TableGroupBy<K> implements ExecutionStep<KGroupedTable<Struct, GenericRow>> {
```

Comment:
Like others, I think we can drop the generic `K` from this type and just use `?`. `K` is the generic type of the source for this step, it doesn't make sense to have this on the type itself.